plugins/out_stackdriver/stackdriver_conf.c

```diff
@@ -281,6 +308,32 @@ struct flb_stackdriver *flb_stackdriver_conf_create(struct flb_output_instance *
         ctx->resource = flb_sds_create(FLB_SDS_RESOURCE_TYPE);
     }
 
+    flb_kv_init(&ctx->resource_labels);
+
+    /*
+     * Load monitored resource labels from config files.
+     * Labels that don't match for a particular resource type will be
+     * rejected by the API.
+     *
+     * For a list of valid labels, see:
+     * https://cloud.google.com/logging/docs/api/v2/resource-list
+    */
+    mk_list_foreach(labels, &ins->properties) {
+        kv = mk_list_entry(labels, struct flb_kv, _head);
+        if (strncasecmp(kv->key, rlabel_prefix, rlabel_prefix_len) == 0 &&
```

Comment:
you can use flb_sds_cmp() to make it easier:

https://github.com/fluent/fluent-bit/blob/master/include/fluent-bit/flb_sds.h#L81

Reply:
Done.